hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -356,7 +356,9 @@ private StorageContainerManager(OzoneConfiguration conf,
     // Authenticate SCM if security is enabled, this initialization can only
     // be done after the metadata store is initialized.
     if (OzoneSecurityUtil.isSecurityEnabled(conf)) {
-      initializeCAnSecurityProtocol(conf, configurator);
+      if (!securityConfig.isCustomCAEnabled()) {
+        initializeCAnSecurityProtocol(conf, configurator);
+      }
```

Comment:
What happens when custom root CA is enabled? Where do we initialize?

Reply:
Right now, we don't have to initialize the root CA certificate server or intermediate certificate servers for SCM since SCM doesn't have to sign any certificate when the external root CA is enabled. But, the security protocol server will have to be initialized and started for persisting the external root CA-signed certificates of all the components. This will be initialized via https://issues.apache.org/jira/browse/HDDS-6032